PKGBUILD

```
@@ -8,12 +8,18 @@ arch=('x86_64')
 url="https://github.com/Boeing/config-file-validator"
 license=('Apache 2.0')
 depends=('glibc')
-makedepends=('go>=1.21')
-source=("$pkgname-$pkgver.tar.gz::$url/archive/refs/tags/v$pkgver.tar.gz")
+makedepends=('go>=1.21' 'git' 'sed')
+source=("git+https://github.com/Boeing/config-file-validator.git")
 sha256sums=('SKIP')
+md5sums=('SKIP')
+
+pkgver() {
+  cd "$srcdir/$pkgname"
+  git describe --tags | sed 's/^v//;s/-.*//'
```

Can simplify the sed using the `--abbrev=0` arg:

```suggestion
  git describe --tags --abbrev=0 | sed 's/^v//'
```

---

@kehoecj Updated. Thanks for pointing out.